test/test_datapipe.py

```diff
@@ -1577,6 +1577,10 @@ def test_shuffler_mapdatapipe(self):
         shuffler_dp = dp.map.Shuffler(input_dp2, indices=['a', 'b', 'c', 'd', 'e'])
         self.assertEqual(set(range(1, 6)), set(shuffler_dp))
 
+        #  Functional Test: deactivate shuffling via set_shuffle
+        shuffler_dp = dp.iter.Shuffler(input_dp1).set_shuffle(False)
+        self.assertEqual(list(shuffler_dp), list(input_dp1))
```

Comment:
I just realized this test function is about `dp.map.Shuffler` (which I did not know existed), but the one we're interested in is  `dp.iter.Shuffler`. LMK where it would make more sense to have this test?

Reply:
Here:

https://github.com/pytorch/pytorch/blob/a8e81ccad8c22e805c84a11f661f7cb643f606dd/test/test_datapipe.py#L1384